Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/MetsKitodoReader.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataeditor;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.net.URI;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.stream.StreamSource;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.dataformat.metskitodo.Mets;
+import org.xml.sax.InputSource;
+
+/**
+ * This class provides methods for reading sources to Mets objects.
+ */
+class MetsKitodoReader {
+
+    private static final Logger logger = LogManager.getLogger(MetsKitodoReader.class);
+
+    /**
+     * Reads an old mets-mods-goobi formated xml file by transforming it to the
+     * current used mets-kitodo format.
+     * 
+     * @param xmlFile
+     *            The file as URI object.
+     * @return The Mets object in mets-kitodo format.
+     */
+    static Mets readUriToMetsFromOldFormat(URI xmlFile) throws TransformerException, IOException, JAXBException {
+        URI xslFile = URI.create("./src/main/resources/xslt/MetsModsGoobi_to_MetsKitodo.xsl");
```

Comment:
Is this hard coded path indented? This will probably not work outside a development system. This mentioned file should be searched in and used from `xslt` folder of the Kitodo.Production installation.